app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PostsController < ApplicationController
+  before_action :authenticate_user!
+
```

Comment:
- [ ] In order to use  **CanCanCan** for Post authorization in `Post Controller`, add this line `load_and_authorize_resource` on the top of your controller 👇🏻 

   ```ruby
     class PostsController < ApplicationController
       load_and_authorize_resource
   
       ...
     end
   ```